spec/fixtures/teachers.yml

```diff
@@ -4,7 +4,7 @@
 #
 #  id                                :bigint           not null, primary key
 #  admin                             :boolean          default(FALSE)
-#  application_status                :string           default("Pending")
+#  application_status                :string           default("pending")
```

Comment:
I thought we decided that it was going to be capitalized? Not a big deal.

Reply:
Yeah not sure, what happened. I mentioned it in my pull request comments, but I must've made a pull request of an older version that was updated later or something. I'll edit out the changes?